Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -183,8 +183,9 @@
         "JenkinsArtifactDetailsParsingError": "Parsing ArtifactDetails failed with an error %s",
         "SuccessfullyDownloadedCommitsAndWorkItems": "Commits and WorkItems Downloaded successfully",
         "CommitsAndWorkItemsDownloadFailed": "Downloading Jenkins Commits and WorkItem failed with an error: %s",
-        "InvalidBuildIndex": "Invalid build index found",
-        "JenkinsNoCommitsToFetch": "Deploying same build. Nothing to fetch",
-        "JenkinsRollbackDeployment": "Handling rollback deployment, start build (%s) is greater than end build (%s)"
+        "CannotFindBuilds": "Cannot find builds",
+        "JenkinsNoCommitsToFetch": "Start and end build Ids are same. Commits and workitems will not be downloaded.",
+        "JenkinsRollbackDeployment": "Handling rollback deployment, start build (%s) is greater than end build (%s)",
```

Comment:
Let's remove "deployment" from here as well.
Let's have a generic message: Downloading commits and workitems between builds %s and %s. 

Let's not have a separate message for rollback. Just add a debug saying start is > end